packages/server-reason-react-ppx/server_reason_react_ppx.ml

```diff
@@ -10,6 +10,12 @@ let shared_folder_prefix = ref None
 let repo_url = "https://github.com/ml-in-barcelona/server-reason-react"
 let issues_url = Printf.sprintf "%s/issues" repo_url
 
+let match_substring string substring =
+  try
+    Str.search_forward (Str.regexp substring) string 0 |> ignore;
```

Comment:
Can you use Str.regexp_string instead of regexp to not have any surprising issues?

Reply:
Sure